lesson_19/pop_victor/tema19-CM-VictorPOP.py

```diff
@@ -0,0 +1,62 @@
+# Implement a context manager called just_some_exceptions that will handle
+# KeyError, IndexError by printing a message and let any other exceptions
+# propagate outside of the context manager.
+
+
+from contextlib import contextmanager
+
+
+class Just_Some_Exceptions_Class:
+
+    def __enter__(self):
+        pass
+
+    def __exit__(self, exc_type, exc_value, traceback):
+        if exc_type is IndexError:
+            print('IndexError found with class')
+            return True
+        elif exc_type is KeyError:
+            print('KeyError found with class')
+            return True
+        elif exc_type:
+            print('other error found with class')
+            return True
```

Comment:
the other errors should propagate. If we return True the exceptions will be suppressed